src/instructlab/configuration.py

```diff
@@ -314,13 +314,6 @@ class _generate(BaseModel):
         default_factory=lambda: DEFAULTS.DATASETS_DIR,
         description="Directory where generated datasets are stored.",
     )
-    # TODO: remove this? It's not used anywhere, was removed by 19b9f4794f79ef81578c00c901bac3ee9db8c046
-    # related issue: https://github.com/instructlab/instructlab/issues/2261
-    seed_file: StrictStr = Field(
-        description="Path to seed file to be used for generation.",
-        default_factory=lambda: DEFAULTS.SEED_FILE,
-        deprecated=True,
-    )
```

Comment:
Why are we doing this in this PR?

Reply:
The docs build throws warning when building the pydantic documentation because this field wasn't used anywhere in the codebase.

I could have added a flag to the docs build that says don't fail the docs builds because of warnings, but I thought it would be better to knock this issue out and have the infra in place to make sure the code base doesn't have unused config variables for the future.

Reply:
Fair enough! I'm fine with this